crates/evm/evm/src/executors/invariant/mod.rs

```diff
@@ -492,8 +494,14 @@ impl<'a> InvariantExecutor<'a> {
     ) -> Result<(SenderFilters, FuzzRunIdentifiedContracts)> {
         let targeted_senders =
             self.call_sol_default(to, &IInvariantTest::targetSendersCall {}).targetedSenders;
-        let excluded_senders =
+        let mut excluded_senders =
             self.call_sol_default(to, &IInvariantTest::excludeSendersCall {}).excludedSenders;
+        // Extend with default excluded addresses - https://github.com/foundry-rs/foundry/issues/4163
+        excluded_senders.extend([
+            CHEATCODE_ADDRESS,
+            HARDHAT_CONSOLE_ADDRESS,
+            DEFAULT_CREATE2_DEPLOYER,
```

Comment:
We should consider having a const for these addresses since we reference them individually in a lot of places